include/bgm.h

```diff
@@ -0,0 +1,6 @@
+#ifndef BGM_H
+#define BGM_H
+
+#define NA_BGM_STOP 0x100000FF
+
+#endif
```

Comment:
I think having this macro in is fine for now, but I suspect it might be replaced by a more general macro. I can't say for certain until I get to decomping `code_801A7B10.c` and have a better understanding of how these commands work. It may warrant further discussion, but it'll wait until `code_801A7B10.c` is further along

Reply:
I took it from OoT (that's the whole header). Do you think I should remove it?

Reply:
No you can leave it in, it'll help with audio docs later on

Reply:
The OoT header `bgm.h` was last edited on the initial commit of the repo, that's pretty funny. But yeah not recommending any change here, only bringing this up as a setup for future discussions and to be aware that this is being thought of